cmd/podman/images/push.go

```diff
@@ -75,6 +75,7 @@ func init() {
 func pushFlags(cmd *cobra.Command) {
 	flags := cmd.Flags()
 
+	flags.BoolVar(&pushOptions.All, "all", false, "push all of the images referenced by the manifest list")
```

Comment:
Docker has an `-a/--all-tags` option that pushes all the tags.  I'm leery about naming this `--all` especially if we shorten it to `-a`.  They also have a `manifest push` command.  Should we make "all" push tags and manifests?